src/applications/travel-pay/components/SmocRadio.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { VaRadio } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+export default function SmocRadio({
```

Comment:
"_Welcome back to S-M-O-C _**The SMOC**_ radio, we've got some great hits lined up for you in the next hour!"

Reply:
"The _smoooth_ sounds of SMOC"